src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/AccountProcessor.kt

```diff
@@ -327,14 +328,25 @@ internal class V4AccountProcessor(
         return modified
     }
 
-    internal fun receivedTransfers(
+    fun processTransfers(
+        existing: InternalAccountState,
+        payload: List<IndexerTransferResponseObject>?,
```

Comment:
can we just skip this call entirely if null?

Reply:
We can.. but the subaccount processor already handles it for us.